mteb/abstasks/TaskMetadata.py

```diff
@@ -259,3 +259,9 @@ def get_script(lang: str) -> str:
                 get_script(lang) for langs in self.eval_langs.values() for lang in langs
             )
         return set(get_script(lang) for lang in self.eval_langs)
+
+    def is_filled(self) -> bool:
+        """Check if all the metadata fields are filled."""
+        return all(
+            getattr(self, field_name) is not None for field_name in self.model_fields
```

Comment:
@HLasse it might be ideal to update the taskmetadata to specify what the values should be e.g. empty list for domains or empty string for "no citations".

Reply:
Sure thing!

Reply:
Should it be specified somewhere that `None` is discouraged and only meant for historic datasets? Could be either in the adding a dataset markdown or `TaskMetadata`?

Reply:
If not, should be ready to merge now

Reply:
I think the current is nice - we can always update dater